tests/providers/amazon/aws/operators/test_comprehend.py

```diff
@@ -0,0 +1,163 @@
+            language_code=LANGUAGE_CODE,
+            data_access_role_arn=ROLE_ARN,
+            **op_kw,
+        )
+
+        assert comprehend_base_op.aws_conn_id == (aws_conn_id if aws_conn_id is not NOTSET else "aws_default")
+        assert comprehend_base_op.region_name == (region_name if region_name is not NOTSET else None)
+
+    @mock.patch.object(ComprehendBaseOperator, "hook", new_callable=mock.PropertyMock)
+    def test_initialize_comprehend_base_operator_hook(self, comprehend_base_operator_mock_hook):
+        comprehend_base_op = ComprehendBaseOperator(
+            task_id="comprehend_base_operator",
+            input_data_config=INPUT_DATA_CONFIG,
+            output_data_config=OUTPUT_DATA_CONFIG,
+            language_code=LANGUAGE_CODE,
+            data_access_role_arn=ROLE_ARN,
+        )
+        mocked_hook = mock.MagicMock(name="MockHook")
+        mocked_client = mock.MagicMock(name="MockClient")
+        mocked_hook.conn = mocked_client
+        comprehend_base_operator_mock_hook.return_value = mocked_hook
+        assert comprehend_base_op.client == mocked_client
+        comprehend_base_operator_mock_hook.assert_called_once()
+
+
+class TestComprehendStartPiiEntitiesDetectionJobOperator:
+    JOB_ID = "random-job-id-1234567"
+    MODE = "ONE_DOC_PER_LINE"
+    JOB_NAME = "TEST_START_PII_ENTITIES_DETECTION_JOB-1"
+    DEFAULT_JOB_NAME_STARTS_WITH = "start_pii_entities_detection_job"
+
+    @pytest.fixture
+    def mock_conn(self) -> Generator[BaseAwsConnection, None, None]:
+        with mock.patch.object(ComprehendHook, "conn") as _conn:
+            _conn.start_pii_entities_detection_job.return_value = {"JobId": self.JOB_ID}
+            yield _conn
+
+    @pytest.fixture
+    def comprehend_hook(self) -> Generator[ComprehendHook, None, None]:
+        with mock_aws():
+            hook = ComprehendHook(aws_conn_id="aws_default")
+            yield hook
+
+    def setup_method(self):
+        self.operator = ComprehendStartPiiEntitiesDetectionJobOperator(
+            task_id="start_pii_entities_detection_job",
+            input_data_config=INPUT_DATA_CONFIG,
+            output_data_config=OUTPUT_DATA_CONFIG,
+            data_access_role_arn=ROLE_ARN,
+            mode=self.MODE,
+            language_code=LANGUAGE_CODE,
+            start_pii_entities_kwargs={"JobName": self.JOB_NAME},
+        )
+        self.operator.defer = mock.MagicMock()
+
+    def test_init(self):
+        assert self.operator.input_data_config == INPUT_DATA_CONFIG
+        assert self.operator.output_data_config == OUTPUT_DATA_CONFIG
+        assert self.operator.data_access_role_arn == ROLE_ARN
+        assert self.operator.mode == self.MODE
+        assert self.operator.language_code == LANGUAGE_CODE
+        assert self.operator.start_pii_entities_kwargs.get("JobName") == self.JOB_NAME
+
+    @pytest.mark.parametrize(
+        "start_pii_entities_kwargs",
+        [
+            pytest.param(
```

Comment:
Did you plan to add more parameters and not get around to it? If there is just one test case then you don't need the parametrization right?

Reply:
Updated now.